.github/workflows/build.yml

```diff
@@ -0,0 +1,61 @@
+name: Test
```

Comment:
As it's in the `build.yml` we could rename this to `Build` and shorten the jobs to be `Android` and `iOS`?

Reply:
fix: [beebd20](https://github.com/evva-sfw/abrevva-flutter/pull/4/commits/beebd20b665e992166676a79aecc953761b42e12)